docs/tutorials/adding-a-module-to-your-runtime.md

```diff
@@ -134,52 +145,86 @@ error: aborting due to previous error
 But since you did not forget, you should be able to sanity check that everything compiles correctly with:
 
 ```bash
-// Build Wasm binaries with `no_std`
-./scripts/build.sh
-
-// Build native binaries with `std`
 cargo build --release
 ```
 
-## Adding the Contract Module
+## Adding the Contracts module
+
+Now that we have successfully imported the Contracts module crate, we need to add it to our Runtime. The first thing we will add to our runtime is the Gas type.
+
+**lib.rs**
+```rust
+//! The Substrate Node Template runtime. This can be compiled with `#[no_std]`, ready for Wasm.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+// `construct_runtime!` does a lot of recursion and requires us to increase the limit to 256.
+#![recursion_limit="256"]
+
+// Make the WASM binary available.
+#[cfg(feature = "std")]
+include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
+
+use rstd::prelude::*;
+...
```

Comment:
For omitting, I do prefer the way in [Rust books](https://doc.rust-lang.org/book/ch15-05-interior-mutability.html) with `// --snip--`.

Reply:
Good idea. I'll start using it.